<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SN</class>
 <widget class="QGroupBox" name="SN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(00,00,00);
color: #FFFFFF;
}
QPushButton{
	background-color: black;
}
 QPushButton:pressed {
     background-color: rgb(224, 0, 0);     
}
QPushButton:hover {
     background-color: rgb(100, 30, 30);
}
QPushButton#startButton:disabled:checked {
	color: gray;
}
QPushButton#startButton:disabled:checked {
	background-color: #ffaaaa;
}
QPushButton#startButton:disabled:!checked {
	background-color: lightGreen;
}

QMessageBox {
    background-color: rgb(00,00,00);
    color: #FFFFFF;
}
QMessageBox QLabel {
    color: #FFFFFF;
}
QMessageBox QPushButton {
    background-color: rgb(50,50,50);
    color: #FFFFFF;
}
QMessageBox QPushButton:hover {
    background-color: rgb(100, 30, 30);
}
QMessageBox QPushButton:pressed {
    background-color: rgb(224, 0, 0);
}</string>
  </property>
  <widget class="QLineEdit" name="globalY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="globalZ">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="globalX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localX">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â†”</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localZ">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="localY">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveStageXY">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/move_xy0.png</normaloff>resources/move_xy0.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>globalX</tabstop>
  <tabstop>globalY</tabstop>
  <tabstop>globalZ</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>localX</tabstop>
  <tabstop>localY</tabstop>
  <tabstop>localZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
